src/typeDefs/types.ts

```diff
@@ -102,7 +113,7 @@ export const types = gql`
     title: String!
     description: String!
     startDate: Date!
-    endDate: Date!
+    endDate: Date
```

Comment:
is this expected to have a nullable endDate?

Reply:
I guess, yes, an event may have a null as endDate. @palisadoes?